SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security and Coordinated Vulnerability Disclosure Policy
+
+This project appreciates and encourages coordinated disclosure of security vulnerabilities. We prefer that you use the GitHub reporting mechanism to privately report vulnerabilities. Under the main repository's security tab, click "Report a vulnerability" to open the advisory form.
+
+If you are unable to report it via GitHub, have received no response after repeated attempts, or have other security related questions, please contact security @ gr-oss.io and mention this project in the subject line.
```

Comment:
Is it intentional that there are spaces around the `@`?

Reply:
Yep -- it's intended to serve as a failsafe in case the repo is harvested for email addresses, though this may not be as big an issue at the moment.

Reply:
OK thanks, I guess it's fairly obvious that people need to remove the spaces so that seems fine